Add Senti_Lex Dataset (#1502)
huggingface/datasets: datasets/senti_lex/README.md

---
annotations_creators:
- expert-generated
language_creators:
- expert-generated
languages:
- af
- an

> - az

Reviewer: can you add the other languages please ?

- **Point of Contact:** [Needs More Information]

### Dataset Summary

This dataset add sentiment lexicons for 81 languages generated via graph propagation based on a knowledge graph--a graphical representation of real-world entities and the links between them

### Supported Tasks and Leaderboards

Sentiment-Classification

### Languages

Afrikaans
Aragonese
Arabic
Azerbaijani
Belarusian
Bulgarian
Bengali
Breton
Bosnian
Catalan; Valencian
Czech
Welsh
Danish
German
Greek, Modern
Esperanto
Spanish; Castilian
Estonian
Basque
Persian
Finnish
Faroese
French
Western Frisian
Irish
Scottish Gaelic; Gaelic
Galician
Gujarati
Hebrew (modern)
Hindi
Croatian
Haitian; Haitian Creole
Hungarian
Armenian
Interlingua
Indonesian
Ido
Icelandic
Italian
Georgian
Khmer
Kannada
Korean
Kurdish
Kirghiz, Kyrgyz
Latin
Luxembourgish, Letzeburgesch
Lithuanian
Latvian
Macedonian
Marathi (Marāṭhī)
Malay
Maltese
Dutch
Norwegian Nynorsk
Norwegian
Polish
Portuguese
Romansh
Romanian, Moldavian, Moldovan
Russian
Slovak
Slovene
Albanian
Serbian
Swedish
Swahili
Tamil
Telugu
Thai
Turkmen
Tagalog
Turkish
Ukrainian
Urdu
Uzbek
Vietnamese
Volapük
Walloon
Yiddish
Chinese

## Dataset Structure

### Data Instances

> 

Reviewer: ```suggestion

An example from the dataset:
```

```
{
"word":"die"

> "sentimen":"negative"

Reviewer: ```suggestion
"sentimen": 0,  # "negative"
```

}
``` 

### Data Fields

- word: one word as a string,

> - sentiment-score: the sentiment classifcation of the word as a string either negative or positive

Reviewer: ```suggestion
- sentiment-score: the sentiment classification of the word as a string either negative (0) or positive (1)
```
